### ISSUE TITLE
Desktop Concept based on SUSI.AI

### ISSUE THREAD

**Issue by meets2tarun** [fossasia/meilix#241]
A more useful configurator and a new desktop concept based on Susi.ai.
@tabesin

**Comment by ms10398** [comment]
@tabesin can we integrate susi terminal client into mellix by default 

Link to the repository - https://github.com/gabru-md/susi-terminal

This was made by @gabru-md and me before

**Comment by ms10398** [comment]
We were thinking to make a complete different terminal window for susi.

which can act as a complete Personal Assistant for the OS.

**Comment by abishekvashok** [comment]
There's susi desktop client as well!

**Comment by ms10398** [comment]
Yeah but our approach was making a complete new client like Sirie or Cortana with its interface of a terminal which have native terminal functionalities and susi query functionalities 

@abishekvashok

**Comment by tabesin** [comment]
Packaged! So it you could install it also on a standard lubuntu. The real point is deep desktop integration and feature parity with Siri as a mvp.

**Comment by ms10398** [comment]
@gabu-md whats your views

**Comment by tabesin** [comment]
@ms10398  Of course. 

We will have to discuss a lot how to do it but what it clear is that we have to take a dedicated repository that builds a deb package, define an API of required device specific fundtionality. Then we should simply look at the competition and find out how to get feature parity. 
https://www.digitalcitizen.life/top-16-most-useful-cortana-commands-questions

**Comment by abishekvashok** [comment]
@tabesin what are the updates currently?
